### ISSUE TITLE
tutorial:  sourceconfig

### ISSUE THREAD

**Issue by Befzz** [xjsframework/xjsframework.github.io#38]
3.6.1811.2318 / 2.9

Error: `SourcePluginWindow class is only available for source plugins`   
Error: `SourcePropsWindow class is only available for source properties`  

Here: https://xjsframework.github.io/tutorials.html#/sourceconfig

Code example from tutorial:
```javascript
// source.html
var xjs = require('xjs');
var sourceWindow = xjs.SourcePluginWindow.getInstance();

sourceWindow.on('save-config', function(configObj) {
  mySource.saveConfig(configObj);
});
```

`xjs.SourcePluginWindow.getInstance();` not working before `xjs.ready()`  resolved

**Comment by dcefram** [comment]
Yeah, you should call `getInstance();` only after `xjs.ready()`. It seems like we failed to update the tutorial when we changed the `SourcePluginWindow`'s constructor logic. Thanks for pointing this out, I'll send a PR to update the tutorial :)
